src/ApiService/ApiService/Functions/Migrations/JinjaToScriban.cs

```diff
@@ -39,11 +39,11 @@ public class JinjaToScriban {
         _log.Info($"Finding notifications to migrate");
 
         var notifications = _context.NotificationOperations.SearchAll()
-            .Select(notification => {
+            .SelectAwait(async notification => {
```

Comment:
non blocking comment 
you have the opportunity here to convert those templates in parallel. depending on the size of the dataset that could speed things up.